app4java/src/main/java/com/nyxei/pollingus/PollingusApplication.java

```diff
@@ -1,13 +1,37 @@
 package com.nyxei.pollingus;
 
-import org.springframework.boot.SpringApplication;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandlers;
+
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
 public class PollingusApplication {
 
 	public static void main(String[] args) {
-		SpringApplication.run(PollingusApplication.class, args);
+		 
+
+		try {
+			HttpRequest request = HttpRequest.newBuilder()
+				.uri(new URI("https://graph.facebook.com/v15.0/102062986090216/messages"))
```

Comment:
The URI parameter should be a constant called whenever you need it.

Reply:
final String URI_rappel = "https://graph.facebook.com/v15.0/102062986090216/messages"